vllm/entrypoints/openai/serving_tokenization.py

```diff
@@ -62,12 +63,12 @@ async def create_tokenize(
         if isinstance(request, TokenizeChatRequest):
             model_config = self.model_config
 
-            conversation: List[ConversationMessage] = []
+            conversation, mm_futures = parse_chat_messages(
+                request.messages, model_config, tokenizer)
 
-            for message in request.messages:
-                result = parse_chat_message_content(message, model_config,
-                                                    tokenizer)
-                conversation.extend(result.messages)
+            if mm_futures:
```

Comment:
QQ: should we warn only once? (I think there's a function like warn_once) it is not going to be spammy?

Reply:
Good point! It's unlikely to be an issue since we have explicitly mentioned that we're only supporting single-image input currently, but yea I agree we should use warn_once here!

Reply:
Actually, the default behaviour already does this (i.e. only print out the first occurrence of the warning).

https://docs.python.org/3/library/warnings.html#the-warnings-filter